File: apps/problems/fruit-into-baskets/boilerplate-full/function.java

    import java.io.*;
    import java.util.*;

    

    //##USER_CODE_HERE##

    public class Main {
      
    public static void main(String[] args) throws IOException {
      BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
      int size_fruits = Integer.parseInt(br.readLine().trim());
        int[] fruits = new int[size_fruits];
        String[] items_fruits = br.readLine().trim().split("\\s+");
        for (int i = 0; i < size_fruits; i++) {
          fruits[i] = Integer.parseInt(items_fruits[i]);
        }
      Solution obj = new Solution();
      int result = obj.totalFruit(fruits);
      System.out.println(result);
      br.close();
    }
      
    }
  

File: apps/problems/fruit-into-baskets/boilerplate-full/function.cpp
#include <iostream>
#include <vector>
#include <string>
#include <sstream>
#include <climits>



//##USER_CODE_HERE


int main() {
    int m_fruits;
std::cin >> m_fruits;
std::vector<int> fruits(m_fruits);

for(int i = 0; i < m_fruits; ++i) {
    std::cin >> fruits[i];
}
    Solution obj;
    int result = obj.totalFruit(fruits);
    std::cout << result << std::endl;
    return 0;
}
